harper-core/dictionary.dict

```diff
@@ -52674,3 +52696,10 @@ write-off/1MS
 # And they will be reviewed
 # Word added using the `just addnoun` command will be added here
 
+Liu/M2 # Surname
+SMTP/M2
+monorepository/M1S
+inherence/M1S
+remap/ABGD
+neurodivergent/PY
+exfiltration/NS
```

Comment:
The first is an adjective so should have the `5` annotation. `P` and `Y` look right for -ness and -ly.
The second is a noun so should have the `1` annotation. `N` adds a nominalization suffix but that's already supplied so would end up doubled like the `re-` prefix before. Again as a noun `M` is always applicable. That would give us:
```
neurodivergent/5PY
exfiltration/1MS
```
Let's use `just getforms` to compare:
```
neurodivergent/5PY
 - neurodivergent
 - neurodivergently
 - neurodivergentness
```
```
exfiltration/NS
 - exfiltration
 - exfiltrations
 - exfiltrationen
```
vs
```
exfiltration/1MS
 - exfiltration
 - exfiltration's
 - exfiltrations
 ```